ctapipe/io/tests/test_dl1eventsource.py

```diff
@@ -18,6 +19,11 @@ def dl1_file():
     return f"{d.name}/testfile.dl1.h5"
 
 
+def test_is_compatible(dl1_file):
```

Comment:
Maybe also test a simtel file to see that `DL1EventSource.is_compatible(simtel_file) == False`